compose/ui/ui-text/src/desktopMain/kotlin/androidx/compose/ui/text/CharHelpers.jvm.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package androidx.compose.ui.text
+
+
+internal actual fun isRtlCodePoint(codePoint: Int): Boolean? {
+    return when (codePoint.getDirectionality()) {
+        CharDirectionality.RIGHT_TO_LEFT,
+        CharDirectionality.RIGHT_TO_LEFT_ARABIC,
+        CharDirectionality.RIGHT_TO_LEFT_EMBEDDING,
+        CharDirectionality.RIGHT_TO_LEFT_OVERRIDE -> true
+
+        CharDirectionality.LEFT_TO_RIGHT,
+        CharDirectionality.LEFT_TO_RIGHT_EMBEDDING,
+        CharDirectionality.LEFT_TO_RIGHT_OVERRIDE -> false
+
+        else -> null
```

Comment:
null is correct value?

Reply:
Yep. For example numbers doesn't point to LTR or RTL. `null` here is kinda unknown/undefined

Reply:
So the boolean value will need to be boxed because of nullability, right?
Maybe we can use int instead? or a value class?

```kotlin
value class CodePointDirection private constructor(val value: Int) {
    companion object {
          val Uknown = CodePointDirection(0)
          val Ltr = CodePointDirection(1)
          val Rtl = CodePointDirection(2)
    }
}
```